bw2io/chemidplus.py

```diff
@@ -25,21 +38,63 @@ def canonical_cas(s):
 
 
 class Multiple(Exception):
-    """Multiple results for given search query."""
-
+    # """Multiple results for given search query."""
```

Comment:
...same question here: [why are you keeping the old docstrings in and commenting them out?](https://github.com/brightway-lca/brightway2-io/pull/169/files#r1141269894)